skimage/feature/tests/test_util.py

```diff
@@ -127,3 +149,68 @@ def test_plot_matched_features(shapes):
         alignment='vertical',
     )
     plt.close()
+
+
+@pytest.mark.skipif(not has_mpl, reason="Matplotlib not installed")
+@pytest.mark.parametrize("matches_color", (42, []))
+def test_plot_matched_features_color_error(matches_color):
+    from matplotlib import pyplot as plt
+    from matplotlib import use
+
+    use('Agg')
+
+    fig, ax = plt.subplots()
+
+    keypoints0 = 10 * np.random.rand(10, 2)
+    keypoints1 = 10 * np.random.rand(10, 2)
+    idxs0 = np.random.randint(10, size=10)
+    idxs1 = np.random.randint(10, size=10)
+    matches = np.column_stack((idxs0, idxs1))
+
+    img0 = np.zeros((10, 10))
+    img1 = np.zeros_like(img0)
+
+    regex = '`matches_color` needs to be a single color '
+    with pytest.raises(ValueError, match=regex):
+        plot_matched_features(
+            img0,
+            img1,
+            ax=ax,
+            keypoints0=keypoints0,
+            keypoints1=keypoints1,
+            matches=matches,
+            matches_color=matches_color,
+        )
+
+
+@pytest.mark.skipif(not has_mpl, reason="Matplotlib not installed")
+def test_plot_matched_features_matplotlib_color_error():
+    # Error is raised from matplotlib itself if we pass a sequence of correct length
```

Comment:
What is "correct length" when we cycle?

Reply:
I updated this comment. We only assume a non-empty sequence which is tested in the other unit test.